web/modules/custom/itk_activity/js/filters.js

```diff
@@ -10,10 +10,31 @@
       var button = $('.js-filters-toggle');
       var filters = $('.js-filters');
 
+      /**
+       * Gets the first query parameter with parameterName.
+       *
+       * @param parameterName
```

Comment:
All most - http://usejsdoc.org/tags-param.html I would like to have type as well as it will tigger validation in the IDE.

`@param {TYPE} NAME`